### PR TITLE
Release 0.0.12

### PR DESCRIPTION
**Pull request by vontell**

### CODE REVIEW

File: docs/changelog.mdx [hunk 1]
```diff
@@ -4,6 +4,43 @@ sidebar_label: 'Changelog'
...
+:::info
+
+This version is a release preview. Our product is in beta and we would love for developers to try it out and provide feedback! Contact
+us at info@regression.gg for dedicated support and access.
+
+:::
+
+**New Platform Features**
+
+- New RGAction attribute within RG Unity SDK to make action definitions easier to make
```
Comment (abeizer) [inline docs/changelog.mdx:18]:
```suggestion
- New RGAction attribute within RG Unity SDK to make actions easier to define
```

File: docs/changelog.mdx [hunk 2]
```diff
@@ -4,6 +4,43 @@ sidebar_label: 'Changelog'
...
+- The RG Unity SDK now includes its own font assets for better interoperability
+- Links to download Zip bots from dashboard are now available
+
+**Bug fixes**
+
+- Fixed issue where the website would freeze in some match loading scenarios
+- Fixed broken links in documentation
+- Fixed issue where Agent Builder code generation would flicker
+- Fixed bug where Unity actions would sometimes be sent to the wrong bots
+- Fixed bug where instant bots would not start when no GitHub account was included in a user account
```
Comment (abeizer) [inline docs/changelog.mdx:32]:
```suggestion
- Fixed bug where instant bots would not start when the user did not have a linked GitHub account
```

**Review by abeizer — APPROVED** [review]
Looks like this contains changes from your GitHub Actions PR. Be careful that this doesn't revert any changes you make in that PR when merging.

**Comment by RG-nAmKcAz** [comment]
@vontell I would do this on a new clean branch from main and submit a new PR

This has unrelated changes interleaved into it

**Comment by vontell** [comment]
@RG-nAmKcAz back to you!

**Review by RG-nAmKcAz — APPROVED** [review]

**Pull request merged**